packages/auth/src/authClient.ts

```diff
@@ -459,6 +451,40 @@ export class AuthClient {
       throw new Error("Parsed state is not of type AuthSessionState");
     return parsedState;
   }
+
+  private async buildAuthUrl(params: LoginWithOauthParams): Promise<string> {
+    const { targetPublicKey } = await this.getTekStamper();
+    const oauthConfig = await this.signerHttpClient.request({
+      route: "signer/v1/prepare-oauth",
+      method: "POST",
+      body: { nonce: getOauthNonce(targetPublicKey) },
+    });
+    return getOauthProviderUrl({
+      oauthParams:
+        params.mode === "redirect" ? { ...params, redirectUrl: "/" } : params, // TO DO: clean up redirect vs popup path
```

Comment:
i think this is totally unrelated to this PR and was already a thing, but i'm curious to know why the `redirectUrl` is always `"/"`, if i'm reading this correctly?